CommMicro.py

```diff
@@ -144,9 +150,16 @@ def getUserVal(self):
           cavNumB=cavNumStr
 
         # Make the path name to be nice
-        LASTPATH=DATA_DIR_PATH+'ACCL_'+liNac+'_'+cmNumStr+cavNumStr[0]+'0'
+#        LASTPATH=DATA_DIR_PATH+'ACCL_'+liNac+'_'+cmNumStr+cavNumStr[0]+'0'
+        LASTPATH=path.join(DATA_DIR_PATH,'ACCL_'+liNac+'_'+cmNumStr+'00')
 
-        return liNac, cmNumStr, cavNumA, cavNumB
+        # get today's date as 2- or 4-char strings
+        year=str(datetime.now().year)

```

Comment:
Just because `datetime.now()` returns something different every time you run it, I'd pull it out as a variable and then just do `variable.year`, `variable.month`, etc